source/funkin/play/stage/Stage.hx

```diff
@@ -454,6 +454,12 @@ class Stage extends FlxSpriteGroup implements IPlayStateScriptedClass implements
       character.cameraFocusPoint.x += stageCharData.cameraOffsets[0];
       character.cameraFocusPoint.y += stageCharData.cameraOffsets[1];
 
+      character.scrollFactor.x += stageCharData.scroll[0];
+      character.scrollFactor.y += stageCharData.scroll[1];
```

Comment:
I don't think these should be additive, perhaps an oversight?

Reply:
lol